VideoLocker/res/layout/create_new_item_layout.xml

```diff
@@ -23,11 +23,12 @@
             android:layout_height="wrap_content"
             android:layout_toEndOf="@+id/create_new_icon_view"
             android:layout_toRightOf="@+id/create_new_icon_view"
-            android:layout_marginStart="10dp"
-            android:layout_marginLeft="10dp"
+            android:layout_marginStart="3dp"
+            android:layout_marginLeft="3dp"
```

Comment:
True, but we should eventually have all dimensions standardized in dime resources, for easier cataloging/editing, and to reduce inconsistencies. Also, maybe this will be used elsewhere later, so it’s better to expose it.
> On 30-Sep-2015, at 6:26 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/create_new_item_layout.xml <https://github.com/edx/edx-app-android/pull/379#discussion_r40792282>:
> 
> > @@ -23,11 +23,12 @@
> >              android:layout_height="wrap_content"
> >              android:layout_toEndOf="@+id/create_new_icon_view"
> >              android:layout_toRightOf="@+id/create_new_icon_view"
> > -            android:layout_marginStart="10dp"
> > -            android:layout_marginLeft="10dp"
> > +            android:layout_marginStart="3dp"
> > +            android:layout_marginLeft="3dp"
> why ? It isn't being used anywhere else
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/379/files#r40792282>.
>